apps/api-gateway/src/issuance/dtos/issuance.dto.ts

```diff
@@ -118,18 +135,22 @@ export class OutOfBandCredentialDto {
     @IsEmail()
     @IsNotEmpty({ message: 'Please provide valid email' })
     @IsString({ message: 'email should be string' })
+    @Transform(({ value }) => value.trim())
```

Comment:
Please also include ToLowercase() for the email.

Reply:
I have included toLowercase() for the email.